diff --git a/drivers/block/sun5i_nand/nfd/mbr.c b/drivers/block/sun5i_nand/nfd/mbr.c
index 142c689..6f19a47 100644
--- a/drivers/block/sun5i_nand/nfd/mbr.c
+++ b/drivers/block/sun5i_nand/nfd/mbr.c
@@ -98,7 +98,7 @@ int mbr2disks(struct nand_disk* disk_array)
 {
 	int part_cnt = 0;
 	int part_index;
-
+#if 0
 	if(_get_mbr()){
 		printk("get mbr error\n" );
 		return part_cnt;
@@ -127,7 +127,12 @@ int mbr2disks(struct nand_disk* disk_array)
 	_free_mbr();
 	PRINT("The %d disk size = %lu\n", part_index - 1, disk_array[part_index - 1].size);
 	PRINT("part total count = %d\n", part_index);
-
+#else
+	part_index = 1;
+	disk_array[0].offset = 0;
+	disk_array[0].size = DiskSize;
+	part_secur[0] = 0;
+#endif
 	return part_index;
 }
 
